pkg/util/exec/exec.go

```diff
@@ -87,16 +88,27 @@ func startCommand(command string, arg ...string) (*exec.Cmd, io.ReadCloser, io.R
 
 func logOutput(stdout, stderr io.ReadCloser) {
 	if stdout == nil || stderr == nil {
-		log.Printf("failed to collect stdout and stderr")
+		logger.Warningf("failed to collect stdout and stderr")
 		return
 	}
 
+	// The logger for this exec package is responsible for logging output from other processes.
+	// Those child processes should appropriately be outputting at the desired global level.  Therefore,
+	// we always log at INFO level here, so that log statements from child procs at higher levels
+	// (e.g., WARNING) will still be displayed.  We are relying on the child procs to output appropriately.
+	if !logger.LevelAt(capnslog.INFO) {
+		rl, err := capnslog.GetRepoLogger("github.com/rook/rook")
+		if err == nil {
+			rl.SetLogLevel(map[string]capnslog.LogLevel{"util/exec": capnslog.INFO})
```

Comment:
will this log level only take effect in child processes if they also use the capnslog package?